src/Qumulo/docs/Update-AzQumuloFileSystem.md

```diff
@@ -0,0 +1,199 @@
+---
+external help file:
+Module Name: Az.Qumulo
+online version: https://learn.microsoft.com/powershell/module/az.qumulo/update-azqumulofilesystem
+schema: 2.0.0
+---
+
+# Update-AzQumuloFileSystem
+
+## SYNOPSIS
+Update a FileSystemResource
```

Comment:
Update does not take effect

Reply:
Customed cmdlet will be covered, new knowledge.